src/custom-styles.css

```diff
@@ -20,6 +21,38 @@
 
 			.Nav {
 				background-color: var(--color-accent-bg-dark);
+				color: var(--color-text-light);
+			}
+			.cross-icon-btn {
+				background-color: var(--color-accent-bg-dark);
+			}
+			.cross-icon {
+				color: var(--color-text-light);
+			}
+			.owner-message {
+				background-color: hsla(000, 0.5%, 20%, 1);
+				color: var(--color-text-dark);
+			}
```

Comment:
This isn't a big deal, but I noticed that on light mode the background color for the message matched the background color for the message. In dark mode there's a slight variation. Just wanted to check to make sure that was intentional. 
<img width="1100" alt="Screenshot 2024-04-05 at 8 38 29 PM" src="https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/97368143/a7152a80-d5ae-4423-8894-02ed87b51e37">

Reply:
We spent a long time trying to figure out how to match it to the background. We could not just use our background-dark css variable color because it has opacity, (that's why the color isn't exactly black but that charcoal color) so the background of that <p> tag that states the owner of the list would be clear which would show the list items behind it, which we didn't want. So we tried to match it as much as possible as much we could(we experimented with RGB and hex and hsla colors and landed with the one we chose as it looked pretty close on my screen. So even though isn't quite as bad on my screen, I do understand that it will vary from monitor to monitor and screen to screen. We are hoping this could just be something we address at a later time.

Reply:
Yeah, I kind of figured it was due to the opacity. It really doesn’t look bad, I just thought I’d double check.